src/esmockLoader.js

```diff
@@ -15,6 +15,9 @@ const urlDummy = 'file:///' + path
   .join(path.dirname(url.fileURLToPath(import.meta.url)), 'esmock.js')
   .replace(/^\//, '');
 
+const isLT1612 = ' 16. 12' > process.versions.node.split('.')
+  .slice(0, 2).map(s => s.padStart(3)).join('.');
```

Comment:
I just like comparing numbers than strings, if you don't want the package node-semver. 😄 
```js
const [major, minor, patch] = process.versions.node.split('.').map(it => +it);

const isLT1612 = major < 16 || (major === 16 && minor < 12);
```

Reply:
@aladdin-add I like this suggestion

Reply:
notes for myself, for tomorrow
```diff
 -// later versions of node v16 include 'node-addons'
 +// later versions of node v14 and v16 include 'node-addons'
```